source/actions3/AMActionSupport.h

```diff
@@ -66,11 +68,11 @@ namespace AMActionSupport
 	}
 
 	/// Helper method that takes an AMControl and a desired setpoint and creates all necessary components.  Caller is responsible for memory.
-	inline AMAction3 *buildControlWaitAction(AMControl *control, double setpoint, double timeout = 10)
+	inline AMAction3 *buildControlWaitAction(AMControl *control, double setpoint, double timeout = 10, AMControlWaitActionInfo::MatchType matchType = AMControlWaitActionInfo::MatchEqual)
```

Comment:
It controls what conditions the control wait action should wait for (current equal target, current < target etc). Previously you had to construct on manually in order to get at this setting, I added it to the action support builder method. 
I also added a matchType for within tolerance.